docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -182,11 +182,12 @@ the action:
     xtrabackup: Transaction log of lsn (5696709) to (5696718) was copied.
     160401 10:25:51 completed OK!
     ```
+    
+*xtrabackup* does not copy the keyring file into the backup directory. To prepare the backup, you must copy the keyring file manually to the backup directory.
```

Comment:
@patrickbirch it is not mandatory to copy to the backup directory. It can be passed via the command line option I mentioned in JIRA
--component-keyring-config (?)

Reply:
It looks as if you should provide a parameter

keyring_args="--component-keyring-config=${keyring_component_cnf}"

Reply:
keyring_args? this is a test variable.

Reply:
the options are always passed as --option-name=x